dataservice/api/biospecimen/schemas.py

```diff
@@ -49,5 +50,7 @@ class Meta(BaseSchema.Meta):
         'sequencing_center': ma.URLFor('api.sequencing_centers',
                                        kf_id='<sequencing_center_id>'),
         'genomic_files': ma.URLFor('api.genomic_files_list',
-                                   biospecimen_id='<kf_id>')
+                                   biospecimen_id='<kf_id>'),
+        'diagnoses': ma.URLFor('api.diagnoses_list',
```

Comment:
Won't this not exist for many entries? Shouldn't it be patched?

Reply:
We would have to use PatchedURLFor if there was ever a chance that `kf_id` would be null. But since `kf_id` refers to the biospecimen itself, it will never be null. Maybe you're thinking of the case for the parent link (i.e. Family and participant, where in the participant's link to family, the family_id/foreign key could be null)?